clean_dataset.py

```diff
@@ -0,0 +1,69 @@
+"""Script to clean the dataset."""
+
+import argparse
+import re
+
+import numpy as np
+from tqdm import tqdm
+
+import utils
+from bs4 import BeautifulSoup
+from nltk.tokenize import WordPunctTokenizer
+
+
+def cleaner(data):
+    """Clean data.
+
+    Args:
+        data (str): string to clean
+
+    Returns:
+        token_clean (str): cleaned string
+    """
+    # Remove HTML Code
+    data_clean = BeautifulSoup(data, features="lxml")
+    data_clean = data_clean.get_text()
+
+    # Remove URL http
+    data_clean = re.sub("https?://[A-Za-z0-9./]+", "", data_clean)
+
+    # Remove mention @
+    data_clean = re.sub(r"@[A-Za-z0-9]+", "", data_clean)
+
+    # Remove hastag #, or other informations, keep only letter
+    data_clean = re.sub("[^a-zA-Z]", " ", data_clean)
```

Comment:
```suggestion
    data_clean = re.sub("#[^a-zA-Z]", " ", data_clean)
```

Reply:
Corrected in https://github.com/XanX3601/deep_sentiment140/pull/5/commits/f1290653db971dbf3796a5cd9bffd3abf692dfb9.